setup.py

```diff
@@ -31,7 +31,8 @@
         "robotframework-requests",
         "escapism",
         "semver>=2,<3",
-        "yq"
+        "yq",
+        "Jinja2"
```

Comment:
Can u add pyyaml ? and wherever yaml.load() is used change it to yaml.safe_load(), this is to avoid this error message
YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.

Reply:
ack will do the changes